From fe94c15da9cb24977ffae1087793488bd81b1700 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Mon, 4 Mar 2024 09:28:59 +0100
Subject: [PATCH 0750/1695] dts: add stm32 frequency monitor to stm32mp131

Get HSE / HSI clocks from DT.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f62926c009cd27407489b7a326b64e6d9edccc23 dts: add stm32 frequency monitor to stm32mp131]
Change-Id: Iee826eced3ede772dfcfa8279aca7ee9404aee8f
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/364245
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/374127
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp131.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp131.dtsi b/core/arch/arm/dts/stm32mp131.dtsi
index 5b1907346..ec5b2614a 100644
--- a/core/arch/arm/dts/stm32mp131.dtsi
+++ b/core/arch/arm/dts/stm32mp131.dtsi
@@ -51,6 +51,8 @@
 	hse_monitor: hse-monitor {
 		compatible = "st,freq-monitor";
 		counter = <&lptimer3 1 1 0 0>;
+		clocks = <&rcc CK_HSE>, <&rcc CK_HSI>;
+		clock-names = "hse", "hsi";
 		status = "disabled";
 	};
 
-- 
2.47.2

